File: Basic_Data_Structures/Bubble Sort/Bubble_Sort.cpp
# include<bits/stdc++.h>
using namespace std;
typedef long long int ll;

void swapping(ll &p,ll &q)
{
    ll temp = p;
    p = q;
    q = temp;
}

int main()
{
    ll n;
    cin >> n;
    ll* arr = new ll[n];
    for(ll i =0;i<n;i++)
    {
        cin >> arr[i];
    }
    for(ll i=0;i<n;i++)
    {
        for(ll j=i+1;j<n;j++)
        {
            if(arr[i]>arr[j])
            {
                swapping(arr[i],arr[j]);
            }
        }
    }
    for(ll i =0;i<n;i++)
    {
        cout << arr[i] << " ";
    }
    return 0;
}

File: Basic_Data_Structures/Bubble Sort/bubble_sort.py
# definning the function to swap two values, that retyrns void
def swapping(arr,p,q):
    temp = arr[p]
    arr[p] = arr[q]
    arr[q] = temp

# taking input from the user
arr = list(map(int,input().split()))
p = len(arr)
count = p

# main algorithm
for i in range(p):
    for j in range(i,p):
        if(arr[i]>arr[j]):
            swapping(arr,i,j)

# printing out the sorted array
print(arr)
